test/chain-specific-spokepools/Arbitrum_SpokePool.ts

```diff
@@ -24,7 +24,7 @@ describe("Arbitrum Spoke Pool", function () {
     l2GatewayRouter = await createFake("L2GatewayRouter");
 
     arbitrumSpokePool = await (
-      await getContractFactory("Arbitrum_SpokePool", { signer: owner })
```

Comment:
this syntax was actually redundant all over the place and can be removed without impacting anything.